### ISSUE TITLE
The GitHub About description still says Angular 6

### ISSUE THREAD

**Issue by jamesanderson9182** [linnovate/mean#2029]
Hey there, browsing repos at the moment and spotted this. 

The GitHub About description still says Angular 6. I see there was an update done to Angular 13.  I also saw when browsing the GitHub angular topic e.g. https://github.com/topics/angular I assume both places use the same description. 

<img width="1280" alt="angular 6 description" src="https://user-images.githubusercontent.com/11311806/144515749-e158456c-d710-479c-8457-6ef52b03492f.png">

**Comment by Tariqu** [comment]
Yes, I have seen that too. and I think this could only be updated by mentainer
